packages/components/src/public/QueryModel/withQueryModels.tsx

```diff
@@ -56,6 +56,56 @@ export function withSearchParams<T>(Component: WithSearchParamsComponent<T>): Co
     return Wrapped;
 }
 
+function columnHasFilter(fieldKey: string, filters: Filter.IFilter[]): boolean {
+    fieldKey = fieldKey.toLowerCase();
+    return filters.some(filter => filter.getColumnName().toLowerCase() === fieldKey);
```

Comment:
Consider being defensive here of `filters` not being defined.

Reply:
No, it is a bug if filters is undefined. We have gone out of our way to ensure the filters array is always defined, and I'm not going to budge on that. If filters is ever undefined, and causes problems here, the fix will be to ensure that filters are always defined.